.github/workflows/release.yml

```diff
@@ -19,18 +19,17 @@ jobs:
       - uses: actions/checkout@v2
 
       - id: tag
-        uses: freckle/haskell-tag-action@main
+        uses: freckle/haskell-tag-action@v1
         with:
           package-yaml: ${{ matrix.package }}/package.yaml
           tag-prefix: ${{ matrix.package }}-v
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
       - if: steps.tag.outputs.tag
-        uses: freckle/stack-upload-action@main
+        uses: freckle/stack-upload-action@v2
         with:
           working-directory: ${{ matrix.package }}
-          pvp-bounds: lower
         env:
           HACKAGE_API_KEY: ${{ secrets.HACKAGE_UPLOAD_API_KEY }}
-          STACK_YAML: ../stack.yaml
+          STACK_YAML: ../stack-lts-12.26.yaml
```

Comment:
Oh interesting, this is a simple way to get good lower bounds!

Reply:
Yes! And our action will do it automatically normally. The only reason I have to do this is because it's a multi-package repository so the stack-yamls aren't in the current directory when building.